### ISSUE TITLE
Thermal comfort entities display all decimal digits

### ISSUE THREAD

**Issue by Motik68** [dolezsa/thermal_comfort#287]
Hi,

Since 2.1.1 all my thermal comfort sensors display their all decimal digits whatever precision I configure for them:

![image](https://user-images.githubusercontent.com/22507182/226209058-66f5201e-d91c-4e9c-8e74-6d1fd92e1475.png)

![image](https://user-images.githubusercontent.com/22507182/226209095-0732cd57-538d-4727-b92f-4b735b60a8b2.png)

![image](https://user-images.githubusercontent.com/22507182/226209169-0f03b372-a8b4-4f97-9210-a5209467cfa6.png)

Any idea why?

**Comment by rautesamtr** [comment]
https://developers.home-assistant.io/blog/2023/02/08/sensor_presentation_rounding/

Are you using a template function to display the value? Otherwise the frontend should handle the rounding.
Developer States will allways show the full decimals.

**Comment by jossiee** [comment]
same issue here!

![image](https://user-images.githubusercontent.com/8819339/226599821-04bff9a8-3e50-45de-868a-7c3e1c9aea8d.png)

**Comment by Motik68** [comment]
> https://developers.home-assistant.io/blog/2023/02/08/sensor_presentation_rounding/
> 
> Are you using a template function to display the value? Otherwise the frontend should handle the rounding. Developer States will allways show the full decimals.

I didn't use a template, but your answer made me wonder if the unexpected behaviour could be due to the cards I used not handling yet the new rounding properties. I did a test with a simple entity card and all was fine with the correct number of digits 👍

So the thermal comfort add-on works as expected. I will close this issue, and open one for each of the front end add-ons in which I do not get the correct rounding.
